docs/modules/operation/pages/deep-dive/user-management/user-config.adoc

```diff
@@ -19,7 +19,7 @@ The `rtc` user profile is used for the communication of the Real-Time Console on
 == Create a user
 
 . Log in as a user with admin permissions.
-. Click the gear icon at the top-right of the screen.
+. Click the *Gear* icon at the top-right of the screen.
```

Comment:
It's not labelled "gear" so should we be bolding it?

Reply:
We have been bolding it in other documents, along with other icons. I think we should stick with it, since it's an element of the UI that we're directing the user to click on.